app/views/tools/_call.html.erb

```diff
@@ -119,10 +119,8 @@
           </div>
 
           <p style="text-align: center;">
-            <% if ask_for_newsletter_signup? %>
             <br />
             <strong><a href="/register">Sign up</a> for an EFF Action account and save your information for next time.</strong>
```

Comment:
It still makes sense to hide this for logged-in users.

Reply:
Good point